.github/workflows/check-orphaned-issues.yml

```diff
@@ -0,0 +1,84 @@
+# type: Project Management
+# owner: @camunda/monorepo-devops-team
+---
+name: Check for Orphaned Issues
+
+on:
+  schedule:
+    # Run every Friday at 9:00 AM UTC
+    - cron: '0 9 * * 5'
+  pull_request:
+    paths:
+      - .github/workflows/check-orphaned-issues.yml
+      - .github/scripts/check-orphaned-issues.sh
+  workflow_dispatch:
+
+env:
+  GHA_BEST_PRACTICES_LINTER: enabled
+
+jobs:
+  check-orphaned-issues:
+    permissions: {}
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Import Secrets
+        id: secrets
+        uses: hashicorp/vault-action@4c06c5ccf5c0761b6029f56cfb1dcf5565918a3b # v3.4.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/camunda/ci/camunda CAMUNDA_GITHUB_ORG_PROJECT_TOKEN;
+            secret/data/products/camunda/ci/camunda SLACK_WEBHOOK_TEAM_COREAPPLICATION_LEADERSHIP;
+
+      - name: Check for orphaned issues
+        id: check-issues
+        env:
+          GH_TOKEN: ${{ steps.secrets.outputs.CAMUNDA_GITHUB_ORG_PROJECT_TOKEN }}
+        shell: bash
+        run: |
+          .github/scripts/check-orphaned-issues.sh
+
+      - name: Send Slack notification
+        if: steps.check-issues.outputs.orphaned-issues-found != '0' && github.event_name != 'pull_request'
```

Comment:
why is the check for whether it is a pull request necessary? Is this not already filtered out during search?